cocos/native-binding/index.ts

```diff
@@ -1,4 +1,8 @@
 /* eslint-disable @typescript-eslint/no-namespace */
+
+import { Color } from '../core/math/color';
+import { Vec2 } from '../core/math/vec2';
```

Comment:
```suggestion
import type { Color } from '../core/math/color';
import type { Vec2 } from '../core/math/vec2';
```

we can import type only here